.travis.yml

```diff
@@ -0,0 +1,13 @@
+
+sudo: required
+dist: xenial
+language: python2.7
+
+services:
+  - docker
+
+script:
+  - cd tools/docker/builder
+  - sudo ./image-build.sh
+  - sudo ./build.sh map -n -f MSGLIB=zmq
```

Comment:
-n flag not required
Build with tests - add `BUILD_TESTS=ON`

Reply:
@tomereli I fixed it. Thanks!